### ISSUE TITLE
Research and prototype on Geospatial Database

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#170]
## Description
Decide on geospatial database for ingesting new datasets.

## Acceptance criteria
* Ingest geopackage data
* Ingest geojson data
* working geospatial database prototype
## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Ahsankkhan** [comment]
Detail about Microsoft SQL Server Geospatial feature can be found [here](https://github.com/amosproj/amos2024ss04-building-information-enhancer/wiki/Microsoft-SQL-Server-Geospatial-Features)
